runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -969,6 +970,235 @@ J9::TransformUtil::canFoldStaticFinalField(TR::Compilation *comp, TR::Node* node
    return TR_maybe;
    }
 
+static bool isTakenSideOfAVirtualGuard(TR::Compilation* comp, TR::Block* block)
+   {
+   // First block can never be taken side
+   if (block == comp->getMethodSymbol()->getFirstTreeTop()->getEnclosingBlock())
```

Comment:
Wouldn't it be cheaper to just get the first treetop form the comp?

Reply:
I see there's an API `Compilation::getStartTree()`, but under the hood it's doing the same as getting the first tree from the symbol.

Reply:
I see there's an API `Compilation::getStartTree()` but under the hood it's doing the same thing as getting the first tree from the symbol.

Reply:
I'd prefer to standardize on `comp->getStartTree()` since that is the intent and the most usual form in the compiler.